metatron-app/metatron-discovery
# F#2603 metadata detail add dashboard thumbnail

### Description

add dashboard thumbnail in metadata detail modal
**Related Issue** : 

#2603 

### How Has This Been Tested?

1. Exploration -> data
2. click datasource card
3. check dashboard thumbnail is showing properly
4. check when click dashboard showing dashboard in new window
#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
